IHP/ErrorController.hs

```diff
@@ -210,6 +114,7 @@ postgresHandler exception controller additionalInfo = do
         handlePostgresOutdatedError exception errorText = do
             let ihpIdeBaseUrl = ?context.frameworkConfig.ideBaseUrl
             let title = [hsx|Database looks outdated. {errorText}|]
+            -- @todo: Causes error
```

Comment:
@mpscholten A bunch of surprising errors as I try to split modules.

![image](https://github.com/digitallyinduced/ihp/assets/125707/b7453f05-7704-443d-8064-29d92fead48b)

Reply:
I think again ihp-hsx isn't loaded properly for me. I'll try to grab later `master`

Reply:
vscode is showing me an error, which tells me it's somehow not updated

![image](https://github.com/digitallyinduced/ihp/assets/125707/c52ccd15-3cc2-4ce4-85fb-e1f00290ed82)

Changing `flake.nix` to lastest `master` didn't help, also `rm -rf .devenv .direnv`. 

@mpscholten when you get a chance, can you please check if you see the same error?

Reply:
I did `nix flake update` and the error on VScode is gone. But the compiler error [from above](https://github.com/digitallyinduced/ihp/pull/1736/files#r1248972353) is still there

Reply:
I could not reproduce this error. I've pushed a fix for all the other errors I did hit when running locally. It's up at https://github.com/amitaibu/ihp/pull/3

It seems HSX is not correctly loaded from the ihp-hsx directory. Inside my `ghci` I had to type this to get it working:

```
:set -iihp-hsx
:set -package ghc
:l Test/Main
```